tests/src/Loader/classloader/Casting/castclasspointer.ilproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk.IL">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ReferenceLocalMscorlib>true</ReferenceLocalMscorlib>
```

Comment:
Is ReferenceLocalMscorlib really needed?

Reply:
Sigh, hit merge too quickly. I copied this from some other ilproj. Seems like it started with the original default interface methods ilproj files that came from the prototype branch and I've been mindlessly copying it to ilproj files ever since. I'll make a pull request that sweeps them away.